### ISSUE TITLE
FragPipe-Analyst cannot start

### ISSUE THREAD

**Issue by hnyk-yoshi** [Nesvilab/FragPipe-Analyst-doc#6]
Hi,
![capture1](https://user-images.githubusercontent.com/123217101/213829865-2e831625-d7fd-4170-8064-1e121e8a6572.PNG)
![capture2](https://user-images.githubusercontent.com/123217101/213829868-ea17b153-5b6d-4a94-b373-f8b64809113c.PNG)

I tried to start an analysis on FragPipe-Analyst using http://fragpipe-analyst.nesvilab.org/. when I clicked 'Run', it said 'Disconnected from the server- Reload' (please see the attached pic). So I changed to use https://fragpipe-analyst.org/, but after 'In Progress' disappeared, nothing appeared with the screen grayed out.
I am a new user for FragPipe-Analyst so I hope that I can upload my files properly, but it would be great if you can show me how to solve these issues.
Thanks in advance!

**Comment by hsiaoyi0504** [comment]
Hi, @hnyk-yoshi Could you share the file you are using with me? It might be because of the files you are using are not correctly formatted or some weird bugs. If you feel unsafe about the data publicly, you can send them to my email at yishaio@umich.edu. I will check it soon.

**Comment by hnyk-yoshi** [comment]
Hi @hsiaoyi0504, Thank you for your reply. I just noticed that the latest version of FragPipe has been released- I used the older version of FragPipe so my experimental_annotation.tsv file that I made manually might not be suitable for FragPipe-Analyst. I now use the latest version of FragPipe and a experiment_annotation.tsv automatically generated is working well for FragPipe-Analyst. Thank you very much for your help.
